.github/workflows/initiate_test_deployment.yml

```diff
@@ -0,0 +1,22 @@
+name: Run prefect flow
```

Comment:
Is this the name that will appear in our list of workflows on the "Actions" tab of the repo? If so, suggest you make it it a little more descriptive e.g. "Deploy Prefect test instance"

Reply:
You know - I'm not entirely sure yet. I think you're right though - this name is pitifully nondescript and I will update it to explain exactly what it does, succinctly. I will do this in a PR against our collector branch; this PR is really just to get me something in main so it shows up in the GitHub UI. Thanks for this suggestion!

Reply:
our other workflows have the name in quotes, will it work without being quoted?

```
name: "Applies the migrations to the database"
name: "Build & Deploy CloudFront Cognito Auth"
```

Reply:
@chiaberry wrote:

> our other workflows have the name in quotes, will it work without being quoted?

Thank you for pointing that out! I doubt this would have - great catch. I pushed up a change to quote it and make it more precise in meaning.